ext/doc/Cassandra.php

```diff
@@ -200,204 +200,207 @@ final class Cassandra {
+     * @see \Map::__construct()
      */
     const TYPE_INT = 'int';
 
     /**
      * When using a map, collection or set of type timestamp, all of its elements
      * must be instances of Timestamp.
      *
-     * @see Set::__construct()
-     * @see Collection::__construct()
-     * @see Map::__construct()
+     * @see \Cassandra\Set::__construct()
+     * @see \Collection::__construct()
+     * @see \Map::__construct()
      */
     const TYPE_TIMESTAMP = 'timestamp';
 
     /**
      * When using a map, collection or set of type uuid, all of its elements
      * must be instances of Uuid.
      *
-     * @see Set::__construct()
-     * @see Collection::__construct()
-     * @see Map::__construct()
+     * @see \Cassandra\Set::__construct()
+     * @see \Collection::__construct()
+     * @see \Map::__construct()
      */
     const TYPE_UUID = 'uuid';
 
     /**
      * When using a map, collection or set of type varint, all of its elements
      * must be instances of Varint.
      *
-     * @see Set::__construct()
-     * @see Collection::__construct()
-     * @see Map::__construct()
+     * @see \Cassandra\Set::__construct()
+     * @see \Collection::__construct()
+     * @see \Map::__construct()
      */
     const TYPE_VARINT = 'varint';
 
     /**
      * When using a map, collection or set of type timeuuid, all of its elements
      * must be instances of Timeuuid.
      *
-     * @see Set::__construct()
-     * @see Collection::__construct()
-     * @see Map::__construct()
+     * @see \Cassandra\Set::__construct()
+     * @see \Collection::__construct()
+     * @see \Map::__construct()
      */
     const TYPE_TIMEUUID = 'timeuuid';
 
     /**
      * When using a map, collection or set of type inet, all of its elements
      * must be instances of Inet.
      *
-     * @see Set::__construct()
-     * @see Collection::__construct()
-     * @see Map::__construct()
+     * @see \Cassandra\Set::__construct()
+     * @see \Collection::__construct()
+     * @see \Map::__construct()
      */
     const TYPE_INET = 'inet';
 
     /**
      * The current version of the extension.
      */
-    const VERSION = '1.3.2';
+    const VERSION = '1.4.0';
 
     /**
      * The version of the cpp-driver the extension is compiled against.
      */
-    const CPP_DRIVER_VERSION = '2.9.0';
+    const CPP_DRIVER_VERSION = '2.16.2';
 
     /**
      * Creates a new cluster builder for constructing a Cluster object.
      *
      * @return \Cassandra\Cluster\Builder A cluster builder object with default settings
      */
-    public static function cluster() { }
+    public static function cluster(): \Cassandra\Cluster\Builder
+    {
+    }
 
     /**
      * Creates a new ssl builder for constructing a SSLOptions object.
      *
      * @return \Cassandra\SSLOptions\Builder A SSL options builder with default settings
      */
-    public static function ssl() { }
-
+    public static function ssl(): \Cassandra\SSLOptions\Builder
```

Comment:
This is definitely not php5 compatible

Reply:
I didn't manually edit these files, everything is generated from `generate_doc.php`